src/cli.rs

```diff
@@ -106,6 +115,12 @@ impl Options {
             3 | _ => options.log_level = log::LogLevelFilter::Trace
         }
 
+        if let Some(mut args) = matches.values_of("command") {
+            let command = String::from(args.next().unwrap());
```

Comment:
Actually, it might have been safe already before:

```
$ cargo run -- -e
    Finished dev [unoptimized + debuginfo] target(s) in 0.1 secs
     Running `target/debug/alacritty -e`
error: The argument '-e <command>...' requires a value but none was supplied

USAGE:
    alacritty -e <command>...

For more information try --help
```